test/puppetlabs/puppetdb/scf/storage_test.clj

```diff
@@ -292,6 +293,19 @@
             (is (= (assoc fact-map  "one more" "here")
                    (factset-map "some_certname")))))
 
+        (defn new-facts []
+          (time! (get-storage-metric :add-new-fact) [ "new-fact-time" ]))
+        (testing "replace-facts with only additions"
```

Comment:
We need to change the way we're trying to access the storage-metrics in this test. Looking at the example in the [catalog-duplicates](https://github.com/puppetlabs/puppetdb/blob/6.x/test/puppetlabs/puppetdb/scf/storage_test.clj#L1012) test, I can see that it's accessing the [storage-metrics](https://github.com/puppetlabs/puppetdb/blob/6.x/src/puppetlabs/puppetdb/scf/storage.clj#L172) atom and pulling the metrics out of it once a catalog is submitted. I think we should do something similar in this test. 

We could grab the value out of the new **_:add-new-fact_** metric first, assert it's zero, submit a factset, and then check that the metric was properly updated.